portfolio.js

```diff
@@ -77,124 +77,34 @@ function ClosePopUp() {
-      const btMobWhite = document.querySelector('.btMobWhite');
-      btMobWhite.addEventListener('click', ClosePopUp);
-    } else {
-      const btMobDesktop = document.querySelector('.btMobDesktop');
-      btMobDesktop.addEventListener('click', ClosePopUp);
-    }
-  });
-
-  button4.addEventListener('click', () => {
-    const body = document.querySelector('.body');
-    const PopUp = document.createElement('div');
-    PopUp.classList.add('MobilePopUp');
-    PopUp.innerHTML = createStrProjectMob('button4');
-    body.prepend(PopUp);
-    if (window.innerWidth < 768) {
-      const btMobWhite = document.querySelector('.btMobWhite');
-      btMobWhite.addEventListener('click', ClosePopUp);
-    } else {
-      const btMobDesktop = document.querySelector('.btMobDesktop');
-      btMobDesktop.addEventListener('click', ClosePopUp);
-    }
-  });
-
-  button5.addEventListener('click', () => {
-    const body = document.querySelector('.body');
-    const PopUp = document.createElement('div');
-    PopUp.classList.add('MobilePopUp');
-    PopUp.innerHTML = createStrProjectMob('button5');
-    body.prepend(PopUp);
-    if (window.innerWidth < 768) {
-      const btMobWhite = document.querySelector('.btMobWhite');
-      btMobWhite.addEventListener('click', ClosePopUp);
-    } else {
-      const btMobDesktop = document.querySelector('.btMobDesktop');
-      btMobDesktop.addEventListener('click', ClosePopUp);
-    }
-  });
-
-  button6.addEventListener('click', () => {
-    const body = document.querySelector('.body');
-    const PopUp = document.createElement('div');
-    PopUp.classList.add('MobilePopUp');
-    PopUp.innerHTML = createStrProjectMob('button6');
-    body.prepend(PopUp);
-    if (window.innerWidth < 768) {
-      const btMobWhite = document.querySelector('.btMobWhite');
-      btMobWhite.addEventListener('click', ClosePopUp);
-    } else {
-      const btMobDesktop = document.querySelector('.btMobDesktop');
-      btMobDesktop.addEventListener('click', ClosePopUp);
-    }
-  });
+  const buttonStrings = document.querySelector('#portf').querySelectorAll('button');
+  const countIds = buttonStrings.length;
+  const Ids = [];
+  for (let j = 0; j < countIds; j += 1) {
+    Ids.push(buttonStrings[j].id);
+  }
+  for (let k = 0; k < Ids.length; k += 1) {
+    const button = document.querySelector(`#${Ids[k]}`);
+    button.addEventListener('click', () => {
+      const body = document.querySelector('.body');
+      const PopUp = document.createElement('div');
+      PopUp.classList.add('MobilePopUp');
+      PopUp.innerHTML = createStrProjectMob(`${Ids[k]}`);
+      body.prepend(PopUp);
+      if (window.innerWidth < 768) {
+        const btMobWhite = document.querySelector('.btMobWhite');
+        btMobWhite.addEventListener('click', ClosePopUp);
+      } else {
+        const btMobDesktop = document.querySelector('.btMobDesktop');
+        btMobDesktop.addEventListener('click', ClosePopUp);
+      }
+    });
+  }
```

Comment:
- [ ] Good job implementing changes required by the previous reviewer. But, I think while implementing those change there are somethings you broke as right now all your projects cards are hidden, as result impossible to see the popup, make sure you fix that. And everything should work as expected.
**Here is how looks your portfolio:**
![Screenshot from 2022-12-23 05-06-45](https://user-images.githubusercontent.com/83042742/209263544-11cb2111-1a54-4b46-bc54-3c18e5d78eb3.png)